### PR TITLE
ipam/host-local: add ResolvConf argument for DNS configuration

### PR DESCRIPTION
**Pull request by squeed**
This adds the option `resolvConf` to the host-local IPAM configuration. If specified, the plugin will try to parse the file as a resolv.conf(5) type file and return it in the DNS response.

### CODE REVIEW

**Comment by squeed** [comment]
I'm currently not happy with the name - suggestions & bikeshedding encouraged.

File: Documentation/host-local.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ It stores the state locally on the host filesystem, therefore ensuring uniquenes
 * `rangeEnd` (string, optional): IP inside of "subnet" with which to end allocating addresses. Defaults to ".254" IP inside of the "subnet" block.
 * `gateway` (string, optional): IP inside of "subnet" to designate as the gateway. Defaults to ".1" IP inside of the "subnet" block.
 * `routes` (string, optional): list of routes to add to the container namespace. Each route is a dictionary with "dst" and optional "gw" fields. If "gw" is omitted, value of "gateway" will be used.
+* `resolvConf` (string, optional): Path to a `resolv.conf` file to parse and return for the DNS configuration
```
Comment (dcbw) [inline Documentation/host-local.md]: Should specify this is the path on the host system, not inside the container.

File: plugins/ipam/host-local/README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # host-local IP address manager
 
-host-local IPAM allocates IPv4 and IPv6 addresses out of a specified address range.
+host-local IPAM allocates IPv4 and IPv6 addresses out of a specified address range. Optionally,
+it can return a DNS configuration from a `resolv.conf` file.
```
Comment (dcbw) [inline plugins/ipam/host-local/README.md]: same here, path on host system.

**Comment by dcbw** [comment]
otherwise LGTM.  I don't have a problem with the name.

**Comment by dcbw** [comment]
@squeed want to trivially fix up the docs and then I'll merge?

**Comment by squeed** [comment]
@dcbw updated, thanks!

**Comment by tomdee** [comment]
LGTM - merging.

**Pull request merged**